spec/publisher/updating_transaction_spec.rb

```diff
@@ -0,0 +1,61 @@
+feature "Publishing an updated transaction on Publisher", publisher: true do
+  include PublisherHelpers
+
+  let(:title) { title_with_timestamp }
+  let(:slug) { slug_with_timestamp }
+
+  scenario "Publishing an updated transaction on Publisher" do
+    given_there_is_a_published_transaction_artefact
+    when_i_schedule_downtime_for_now_on_the_transaction
+    and_i_visit_the_transaction_on_gov_uk
+    then_the_downtime_message_is_shown
+  end
+
+  private
+
+  def given_there_is_a_published_transaction_artefact
+    create_transaction
+    @published_url = publish_artefact
+  end
+
+  def when_i_schedule_downtime_for_now_on_the_transaction
+    visit_publisher("/downtimes")
+    click_link(title)
+
+    date = Date.today
+    fill_in_schedule_downtime_form(date)
```

Comment:
Maybe explain the logic for what we're doing with the date and trying to achieve in comment/commit message